.github/workflows/canister-tests.yml

```diff
@@ -237,6 +230,15 @@ jobs:
           echo "now: $commit_now"
 
       - uses: ./.github/actions/bootstrap
+      - name: Install nextest (macos)
+        if: ${{ matrix.os == 'macos-latest' }}
+        run: |
+          curl -LsSf https://get.nexte.st/latest/mac | tar zxf - -C ${CARGO_HOME:-~/.cargo}/bin
```

Comment:
nit: I think you could use a single step by either using `fromJSON` and index with the `os` or use something like [this](https://github.com/actions/runner/issues/409) to produce `mac` or `linux`, depending